clang/include/clang/Sema/Sema.h

```diff
@@ -6767,6 +6767,9 @@ class Sema final : public SemaBase {
     };
     std::optional<InitializationContext> DelayedDefaultInitializationContext;
 
+    /// Whether evaluating an expression for a switch case label.
```

Comment:
supernit: pack it with the contextual bools above, for both layout reasons and because it shows there's prior art

Reply:
Done.